src/plugin.py

```diff
@@ -116,56 +122,56 @@ async def pass_login_credentials(self, credentials: Dict[str, str], _ : List[Dic
         elif login_state == WebpageView.PARANOID_USER:
             return await self._handle_retrieve_rsa_result(credentials)
         elif login_state == WebpageView.PARANOID_ENCIPHERED:
-            return await self._handle_manual_eciphering_result(credentials)
+            return await self._handle_manual_enciphering_result(credentials)
         else:
-            logger.error("Unexpected state in pass_login_credentials")
+            logger.error(f"Unexpected state {login_state:r} in pass_login_credentials")
             raise UnknownBackendResponse()
 
-    async def _handle_login_result(self, credentials: Dict[str, str]) -> Union[Authentication, NextStep]: #todo revert when not testing.
+    async def _handle_login_result(self, credentials: Dict[str, str]) -> Union[Authentication, NextStep]:  # todo revert when not testing.
         self._use_paranoid_login = False
 
         data_or_error = self._view.retrieve_data_regular_login(credentials)
-        print("Got regular login data")
+        logger.debug("Got regular login data")
```

Comment:
print debugs from manually testing using python interactive. can be removed entirely.